DevoxxClientMobile/src/main/java/com/devoxx/views/helper/Util.java

```diff
@@ -114,6 +115,28 @@ public static FloatingActionButton createWebLaunchFAB(Supplier<String> urlSuppli
             });
         });
     }
+    
+    public static void requestRating() {
+        String url = null;
+        if(Platform.isAndroid()) {
+            String packageName = "com.devoxx";
```

Comment:
These hardcoded strings (package name, appId, urls) won't change that much probably, but maybe a remote function is a better option here?

Reply:
Ok